composer.json

```
@@ -9,12 +9,12 @@
         }
     ],
     "require": {
-        "php": "~7.1",
+        "php": "~7.2",
```

I don't think 7.1 support should be dropped as the SDK should still support this version for backward compatibility
```suggestion
        "php": "~7.1",
```

---

Reverted change.